azure-pipelines.yml

```diff
@@ -0,0 +1,237 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+- main
+- synapse
+- refs/heads/synapse
+pr:
+- main
+- synapse
+
+variables:
+  buildConfiguration: 'Release'
+  buildPlatform: 'x64'
+  major: 0
+  minor: 3
+  patch: 0
+  buildnum: $[counter(format('{0}.{1}.{2}', variables['major'], variables['minor'], variables['patch']), 1)]
+  version: $(major).$(minor).$(patch).$(buildnum)
+
+stages:
+- stage: Build
+  pool:
+    vmImage: windows-latest
+  jobs:
+  - job: BuildFhirToCdm
```

Comment:
can we use template to make it more clear?
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops

Reply:
Good idea. I've split the tasks into different templates in the update.